missing_in_hooks.md

```diff
@@ -0,0 +1,19 @@
+# Bugs
```

Comment:
Is this kind of todos? Should we list them somewhere else or version in this markdown?

Reply:
No this is just for reviewers (you, ben) to see what needed to change in the hooks, will be deleted before merge